charts/brigade/values.yaml

```diff
@@ -61,12 +61,9 @@ api:
     type: ClusterIP
     externalPort: 7745
     internalPort: 7745
-  # Configure liveness probes except `httpGet` and the belongings
-  #livenessProbe:
-  #  initialDelaySeconds: 20
-  # Configure readiness probes except `httpGet` and the belongings
-  #readinessProbe:
-  #  initialDelaySeconds: 20
```

Comment:
Probably you've mistakenly removed this block?

Reply:
yes, it's my mistake